core/helpers/EEH_Debug_Tools.helper.php

```diff
@@ -587,8 +593,8 @@ public static function printr(
         }
         $var_name = ucwords(str_replace(array('$', '_'), array('', ' '), $var_name));
         $heading_tag = EEH_Debug_Tools::headingTag($heading_tag);
-        $result = EEH_Debug_Tools::headingSpacer($heading_tag);
-        $result .= EEH_Debug_Tools::heading($var_name, $heading_tag, $margin, $line);
+        // $result = EEH_Debug_Tools::headingSpacer($heading_tag);
```

Comment:
may be remove this?

Reply:
i might reinstate that in the  future so wanted to keep it